target/linux/ath79/dts/ar9330_ziking_cpe46b.dts

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9330.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+        model = "ZiKing CPE46B";
+        compatible = "ziking,cpe46b", "qca,ar9330";
+
+        aliases {
+                serial0 = &uart;
+                label-mac-device = &eth0;
```

Comment:
Is this actually what's on the label?

Reply:
Here is a hi res picture of the label: https://git.lsd.cat/g/openwrt-cpe46b/raw/master/images/label.jpg
Inside the firmware there are multiple references to "ZiKing" and the IANA code: https://oidref.com/1.3.6.1.4.1.37260
There are multiple resellers on Aliexpress and other sites not stating the brand or using their own one. The one in the firmware looked like the actual manufacturer but it's hard to be sure.
Here are a few:

 * https://www.aliexpress.com/item/32974861845.html
 * https://deals.jumia.ci/antenne-wifi-cpe-46b-pid5819548
 * https://m.afzhan.com/chanpin/10318793.html

Reply:
The first address in my ART partition matches the label on the device mainboard, so I guess it's the wrong interface and it should be "wifi0", or isn't it?

Reply:
Well, the "first address in [your] ART partition" is 0x0?
And that's assigned to &eth0?

Reply:
Sorry for the confusion. Yes the address on the label is at 0x0 in the ART partition. In the u-boot log the same address is listed as eth0.

From stock firmware boot log:
```

U-Boot 1.1.413 (Aug 29 2012 - 10:36:47)

AP121-2MB (ar9330) U-boot
DRAM:  32 MB
flash size 8388608, sector count = 128
Flash:  8 MB
In:    serial
Out:   serial
Err:   serial
Net:
eth0: <address at 0x0 in ART>
```